docs/index.md

```diff
@@ -1,10 +1,7 @@
-easy-deposit-properties
+EASY-DEPOSIT-PROPERTIES
```

Comment:
```suggestion
easy-deposit-properties
```
Our style is inconsistent over the modules, using "Manual" in some and lowercase module name in others, but uppercase module name is used nowhere.

Reply:
I'll merge this and update it myself, since the rest looks OK.